org.opentosca.bus/org.opentosca.bus.management.deployment.plugin.script/src/main/java/org/opentosca/bus/management/deployment/plugin/script/ManagementBusDeploymentPluginScript.java

```diff
@@ -25,7 +25,7 @@
  */
 public class ManagementBusDeploymentPluginScript implements IManagementBusDeploymentPluginService {
 
-    static final private String TYPES = "{http://opentosca.org/artifacttypes}Chef, {http://opentosca.org/artifacttypes}Ansible, {http://www.example.com/clartigr/tosca}clartigr, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Chef, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Puppet, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Juju, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}CloudFoundry, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}UnixShell, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}ScriptArtifact";
+    static final private String TYPES = "{http://opentosca.org/artifacttypes}PythonArchiveArtifact_w1-wip1, {http://opentosca.org/artifacttypes}PythonScriptArtifact_w1-wip1, {http://opentosca.org/artifacttypes}Chef, {http://opentosca.org/artifacttypes}Ansible, {http://www.example.com/clartigr/tosca}clartigr, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Chef, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Puppet, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}Juju, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}CloudFoundry, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}UnixShell, {http://docs.oasis-open.org/tosca/ns/2011/12/ToscaBaseTypes}ScriptArtifact";
```

Comment:
Maybe list the available types in separate lines for better readability.
Also, remove the version suffixes. However, I would suggest to create an abstract python Artifact Type and create subtypes of it with versioning.

Reply:
Does the hierarchy stuff even work ?

Reply:
Depends on where the type is checked. However, I'd still remove the `w1-wip` in this case.

Reply:
fixed